types/types.go

```diff
@@ -391,6 +391,9 @@ type SystemContext struct {
 	// === dir.Transport overrides ===
 	// DirForceCompress compresses the image layers if set to true
 	DirForceCompress bool
+
+	// Additional tags when creating or copying an image.
+	Tags []string
```

Comment:
Thanks for the review. Would you be fine with renaming it to `AdditionalTags` as well? I guess this can be interesting for other transports as well, so I am hesitant putting `Docker` in the identifier.

Reply:
I don’t really _like_ it, because the tag format/types differ (e.g. it would be natural to support this also in `oci-archive:`/`oci:`, but their reference annotation format is much less restrictive), so any caller that would actually benefit form sharing the field would pretty much have to give up on validating the input itself (the transports would have to do it using their rules), and it forces us into untyped `string`s.

But as the only user so far, I think it’s up to you  If you do make it generic, please add validation of the string input to the code consuming the `AdditionalTags` field in `docker/archive`, though; it would not do to spend minutes generating gigabytes of data only for the resulting archive to be silently invalid.

Reply:
(Either way, keep the separation of the structure into `===` sections, either add a new one or use the existing “Global configuration overrides” one.

Eventually we may need to figure out something that scales better…)

Reply:
@mtrmac Having thought about it for a while, you convinced me. I'd love to have a generic way of moving tags around, but users of this library will end-up checking any user-input in any case and I don't like the idea of redundantly parsing in the lib. Sadly, there is not a super type for references/tags.